### PR TITLE
[CMP] Update Comscore in CCPA

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Comscore should always be loaded in the CCPA framework.

- [x] add unit tests

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [X] Yes (part of the commercial bundle)

## What is the value of this and can you measure success?

Comescore should run in the whether `state.ccpa.doNotSell` is `true` or `false`.
User consent should still be given for it to run in the TCFv2 framework

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/comscore.spec.js [hunk 1]
```diff
@@ -34,6 +64,35 @@ describe('comscore init', () => {
...
+
+    describe.skip('Framework consent: running on consent', () => {
+        beforeEach(() => {
+            jest.resetAllMocks();
+        });
+
+        it('TCFv2 with consent: runs', () => {
+            onConsentChange.mockImplementation(tcfv2WithConsentMock);
+            init();
+            expect(spyConsent).toBeCalledWith(true);
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/comscore.spec.js:76]: `spyConsent` is never called…

Comment (ioanna0) [reply static/src/javascripts/projects/commercial/modules/comscore.spec.js:76]: what happens if you move the spy inside the test?

Comment (mxdvl) [reply static/src/javascripts/projects/commercial/modules/comscore.spec.js:76]: Creating a `_.resetInit` did the trick, thanks @ioanna0 !

**Review by ioanna0 — APPROVED** [review]
LGTM 👍

**Pull request merged**